debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnection.java

```diff
@@ -360,6 +362,24 @@ public OracleConnection executeLegacy(Operations operations) throws SQLException
         return this;
     }
 
+    public ZoneOffset getDatabaseOffset() {
+        try {
+            final String offset = queryAndMap("SELECT DBTIMEZONE FROM DUAL", rs -> {
+                if (rs.next()) {
+                    return rs.getString(1);
+                }
+                return null;
+            });
+            if (!Strings.isNullOrEmpty(offset)) {
+                return ZoneOffset.of(offset);
+            }
+            return null;
+        }
+        catch (SQLException e) {
+            throw new RuntimeException("Failed to obtain database timezone offset", e);
+        }
+    }
```

Comment:
I don't think we need this any longer?

Reply:
You're correct, I'll remove it in the documentation update commit I'm working on.